[16-SNAPSHOT] Not buildable on ARM 64 Linux

### Describe the bug

It looks like it's using an older version of Node. Newer versions of Node are available for Arm64 Linux,
so it's just a matter of upgrading the Node versions.

Node v22.21.1 and later have the Arm64 binaries, the current v22.11.0 does not:
```
Index of /download/release/v22.11.0/

[../](https://unofficial-builds.nodejs.org/download/release/)
[SHASUMS256.txt](https://unofficial-builds.nodejs.org/download/release/v22.11.0/SHASUMS256.txt)                                     29-Oct-2024 21:34                 608
[node-v22.11.0-headers.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-headers.tar.gz)                       29-Oct-2024 19:07             8780269
[node-v22.11.0-headers.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-headers.tar.xz)                       29-Oct-2024 19:07              533728
[node-v22.11.0-linux-x64-glibc-217.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-linux-x64-glibc-217.tar.gz)           29-Oct-2024 21:32            53613473
[node-v22.11.0-linux-x64-glibc-217.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-linux-x64-glibc-217.tar.xz)           29-Oct-2024 21:34            29626616
[node-v22.11.0-linux-x64-musl.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-linux-x64-musl.tar.gz)                29-Oct-2024 20:18            54479335
[node-v22.11.0-linux-x64-musl.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-linux-x64-musl.tar.xz)                29-Oct-2024 20:20            30320228
```

Node v22.21.1 listing (container arm64 binaries):
```
Index of /download/release/v22.21.1/

[../](https://unofficial-builds.nodejs.org/download/release/)
[SHASUMS256.txt](https://unofficial-builds.nodejs.org/download/release/v22.21.1/SHASUMS256.txt)                                     19-Dec-2025 14:25                1420
[node-v22.21.1-headers.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-headers.tar.gz)                       29-Oct-2025 04:40            10198420
[node-v22.21.1-headers.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-headers.tar.xz)                       29-Oct-2025 04:40              566800
[node-v22.21.1-linux-arm64-musl.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-arm64-musl.tar.gz)              19-Dec-2025 14:23            59316948
[node-v22.21.1-linux-arm64-musl.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-arm64-musl.tar.xz)              19-Dec-2025 14:25            32324560
[node-v22.21.1-linux-armv6l.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-armv6l.tar.gz)                  29-Oct-2025 08:13            51902006
[node-v22.21.1-linux-armv6l.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-armv6l.tar.xz)                  29-Oct-2025 08:15            26550480
[node-v22.21.1-linux-loong64.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-loong64.tar.gz)                 01-Dec-2025 04:19            57513876
[node-v22.21.1-linux-loong64.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-loong64.tar.xz)                 01-Dec-2025 04:21            30417148
[node-v22.21.1-linux-riscv64.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-riscv64.tar.gz)                 18-Dec-2025 19:54            56852766
[node-v22.21.1-linux-riscv64.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-riscv64.tar.xz)                 18-Dec-2025 19:55            31321720
[node-v22.21.1-linux-x64-glibc-217.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-x64-glibc-217.tar.gz)           29-Oct-2025 09:28            57266869
[node-v22.21.1-linux-x64-glibc-217.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-x64-glibc-217.tar.xz)           29-Oct-2025 09:29            31165040
[node-v22.21.1-linux-x64-musl.tar.gz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-x64-musl.tar.gz)                29-Oct-2025 05:55            58165450
[node-v22.21.1-linux-x64-musl.tar.xz](https://unofficial-builds.nodejs.org/download/release/v22.21.1/node-v22.21.1-linux-x64-musl.tar.xz)                29-Oct-2025 05:57            31876148
```


### Reproducer

```
[INFO] --- frontend:1.15.4:install-node-and-yarn (install-node-and-yarn) @ primefaces ---
[INFO] Node v22.21.1 was installed, but we need version v22.11.0
[INFO] Installing node version v22.11.0
[INFO] Downloading https://unofficial-builds.nodejs.org/download/release/v22.11.0/node-v22.11.0-linux-arm64-musl.tar.gz to /home/flowlogix/.m2/repository/com/github/eirslett/node/22.11.0/node-22.11.0-linux-arm64-musl.tar.gz
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.4:install-node-and-yarn (install-node-and-yarn) on project primefaces: Could not download Node.js: Got error code 404 from the server. -> [Help 1]
```

### Expected behavior

Build Success

### PrimeFaces edition

None

### PrimeFaces version

16-SNAPSHOT

### Theme

_No response_

### JSF implementation

Mojarra

### JSF version

N/A

### Java version

N/A

### Browser(s)

_No response_

--- a/primefaces/pom.xml
+++ b/primefaces/pom.xml
@@ -21,7 +21,7 @@
         <resources.dir.compressed>${project.build.directory}/classes/META-INF/resources/primefaces</resources.dir.compressed>
         <mockito.version>5.23.0</mockito.version>
         <sonar.sources>src/main/java,src/main/resources</sonar.sources>
-        <node.version>v22.11.0</node.version>
+        <node.version>v22.21.1</node.version>
         <yarn.version>v1.22.22</yarn.version>
         <frontend.directory>${basedir}/src/main/frontend</frontend.directory>
         <frontend.node.env>development</frontend.node.env>

